.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: test-and-release
+
+on:
+  push:
+    branches: [ main ]
+  release:
+    types:
+      - created
+
+jobs:
+  release:
+    name: Release code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Log in to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Extract tags for Docker images
+        id: docker-meta
+        uses: docker/metadata-action@v3
+        with:
+          images: quay.io/aptible/www-prod
+
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.docker-meta.outputs.tags }}
```

Comment:
This [looks like](https://github.com/docker/metadata-action) it will change the naming convention for Docker repo tags, but I don't see a problem with that.